src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,40 @@
+import React = require('react');
+
+export interface CheckBoxProps {
+    text?: string;
+    value?: boolean;
+    boxIcon?: string;
+    tickIcon?: string;
+    onChange?: (value: boolean) => any;
+}
+
+export const CheckBox: React.SFC<CheckBoxProps> = (props) => (
+    <div data-automation-id="CHECKBOX_ROOT"
```

Comment:
The checkbox should have some styling, albeit minimal. The text is too "stuck" to the box. They shouldn't be bottom aligned, more like base-aligned (the tip of the check mark). No, we don't have a matcher for that. :-)